src/sentry/models/integrations/integration.py

```diff
@@ -32,9 +32,6 @@ def get_active_integrations(self, organization_id: str):
 class Integration(DefaultFieldsModel):
     __include_in_export__ = False
 
-    organizations = models.ManyToManyField(
-        "sentry.Organization", related_name="integrations", through=OrganizationIntegration
-    )
```

Comment:
This looks like it could have some fallout as `integration.organizations` won't work anymore. Were we not using the `organizations` property?

Reply:
We are, there is a bunch more on the backend of this PR I assure you lol.

Reply:
I'm splitting out the larger query changes for this PR because it will be a lot and can be addressed separately.